packages/graphql/tests/integration/issues/487.int.test.ts

```diff
@@ -25,7 +25,7 @@ import Neo4j from "../neo4j";
 import { Neo4jGraphQL } from "../../../src/classes";
 import { UniqueType } from "../../utils/graphql-types";
 
-describe("https://github.com/neo4j/graphql/issues/487", () => {
+describe("https://github.com/neo4j/graphql/issues/487, test on union", () => {
```

Comment:
(non-blocking) consider making 2 tests on the same describe, just to keep the same structure